staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -145,9 +145,13 @@ type GenericAPIServer struct {
 	preShutdownHooksCalled bool
 
 	// healthz checks
-	healthzLock    sync.Mutex
-	healthzChecks  []healthz.HealthzChecker
-	healthzCreated bool
+	healthzLock              sync.Mutex
```

Comment:
healthzReadyzLock, better split it into two.

Reply:
sure.